pyproject.toml

```
@@ -116,6 +121,7 @@ filterwarnings = [
     'ignore:.*datetime.datetime.utcfromtimestamp\(\):DeprecationWarning',
     'ignore:(?s).*Pyarrow:DeprecationWarning',
 ]
+addopts = "--cov"
```

From looking at lost of tracebacks on CIs and locally I'd recommend a couple more options we have in MNE-Python
```suggestion
addopts = "--cov -rfEXs --cov-report= --tb=short"
```
some of which could then be removed from https://github.com/cbrnr/sleepecg/blob/94e83901502aafb545f6e0b4c39d435ccf54b837/.github/workflows/cibuildwheel.yml#L98